packages/plugin/src/Elements/Submission.php

```diff
@@ -86,6 +86,10 @@ public function __construct($config = [])
 
     public function __get($name): mixed
     {
+        if (null === $name) {
```

Comment:
I think it might not be `null`
```suggestion
        if (empty($name)) {
```